python/lsst/rubintv/production/pipelineRunning.py

```diff
@@ -321,10 +382,29 @@ def postProcessIsr(self, quantum):
             outputPath=self.locationConfig.calculatedDataPath,
             binSize=self.locationConfig.binning,
         )
-
         self.log.info(f"Wrote binned postISRCCD for {dRef.dataId}")
 
-    def postProcessCalibrate(self, quantum, processingId):
+        if self.locationConfig.location in ["summit", "bts", "tts"]:  # don't fill ConsDB at USDF
+            try:
+                (expRecord,) = self.butler.registry.queryDimensionRecords("exposure", dataId=dRef.dataId)
```

Comment:
One question: will this work for this one:
```
    def postProcessVisitSummary(self, quantum: Quantum) -> None:
        dRef = quantum.outputs["visitSummary"][0]
        vs = self.cachingButler.get(dRef)
        (expRecord,) = self.butler.registry.queryDimensionRecords("exposure", dataId=dRef.dataId)
```
_i.e._ when something is visit-based rather than exposure based, will `"exposure"` be a valid key in the dataId's records dict? Or does that one still need a query?

Reply:
Oh, and one more, though this is just my curiosity about how you work with mypy: that's annotated to return `DimensionRecord | None` so would you just assert that it's `not None` after every such call? Just wondering if there's a neater way around that. I assume not, but just curious given that I'm new to this world.